gtfs-validator/main/src/main/java/org/mobilitydata/gtfsvalidator/notice/StopTooFarFromTripShapeNotice.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.notice;
+
+import org.mobilitydata.gtfsvalidator.table.GtfsStopTime;
+import org.mobilitydata.gtfsvalidator.table.GtfsTrip;
+
+import com.google.common.collect.ImmutableMap;
+
+public class StopTooFarFromTripShapeNotice extends Notice {
+    public StopTooFarFromTripShapeNotice(String stopId, int stopSequence, String tripId, String shapeId, double tripBufferMeters) {
+        super(ImmutableMap.of(
+            "stopId", stopId,
+            "stopSequence", stopSequence,
+            "tripId", tripId,
+            "shapeId", shapeId,
+            "tripBufferMeters", tripBufferMeters
```

Comment:
If we wish to support visualizations we would need to add more fields. 
We can add it later, so please feel free to add a TODO.

Reply:
Thank you for the advice! Yes, I think that is what we will do later. I have already added the TODO for the file.